sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -979,8 +979,23 @@ object JdbcUtils extends Logging with SQLConfHelper {
       namespace: String,
       comment: String): Unit = {
     val dialect = JdbcDialects.get(options.url)
+    val schemaCommentQuery = if (comment.isEmpty) {
+      comment
+    } else {
+      dialect.getSchemaCommentQuery(namespace, comment)
+    }
     executeStatement(conn, options, s"CREATE SCHEMA ${dialect.quoteIdentifier(namespace)}")
-    if (!comment.isEmpty) createNamespaceComment(conn, options, namespace, comment)
+    if (comment.nonEmpty) executeStatement(conn, options, schemaCommentQuery)
```

Comment:
I guess
```
    val dialect = JdbcDialects.get(options.url)
    executeStatement(conn, options, s"CREATE SCHEMA ${dialect.quoteIdentifier(namespace)}")
    if (comment.nonEmpty) {
      executeStatement(conn, options, dialect.getSchemaCommentQuery(namespace, comment))
    }
```

Reply:
Oh. I forgot it.
I want keep the atomic of create namespace with comment.
If create namespace with comment, MySQL should fails all the operations.

Reply:
I want keep the atomic of create namespace with comment.